crates/sui/src/unit_tests/snapshots/sui__upgrade_compatibility__upgrade_compatibility_tests__deponly.snap

```diff
@@ -0,0 +1,33 @@
+---
+source: crates/sui/src/unit_tests/upgrade_compatibility_tests.rs
+expression: normalize_path(err.to_string())
+---
+error[Compatibility E01002]: type mismatch
+  ┌─ /fixtures/upgrade_errors/deponly_errors_v2/sources/UpgradeErrors.move:4:18
+  │
+4 │ module upgrades::upgrades {
+  │                  ^^^^^^^^ New unexpected struct 'NewStruct'.
+  │
+  = Structs are part of a module's public interface and cannot be changed during an upgrade.
```

Comment:
Should this be "removed or changed" to line up with the new wording you're using elsewhere?